clang/lib/DPCT/AnalysisInfo.cpp

```diff
@@ -4274,6 +4278,26 @@ void CallFunctionExpr::buildCalleeInfo(const Expr *Callee,
     if (auto DRE = dyn_cast<DeclRefExpr>(Callee)) {
       buildTemplateArguments(DRE->template_arguments(),
                              Callee->getSourceRange());
+      if (isa<TranslationUnitDecl>(CallDecl->getDeclContext()) &&
```

Comment:
If the current CallExpr is in the global namespace, `::` doesn't need to be added.